src/ontology/mondo-edit.obo

```diff
@@ -391628,18 +391621,26 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disease {source="Orphanet:399329"}
 subset: orphanet_rare {source="Orphanet:399329"}
 subset: rare
+synonym: "epiphysiolysis capitis femoris" RELATED [OMIM:182260]
```

Comment:
a lot of OMIM synonyms were added as RELATED synonyms but really should be EXACT. Could you please check in OMIM that this is an alternative name, and if so, make this an exact synonym? Thanks!